code/controllers/subsystems/ticker.dm

```diff
@@ -387,6 +387,12 @@ var/datum/controller/subsystem/ticker/SSticker
 	SSjobs.DivideOccupations() // Apparently important for new antagonist system to register specific job antags properly.
 
 	if(!src.mode.can_start())
+		var/list/voted_not_ready = list()
+		for(var/mob/abstract/new_player/player in player_list)
+			if((player.client)&&(!player.ready))
+				voted_not_ready += player.ckey
+		message_admins("The following players voted for [mode.name], but did not ready up: [jointext(voted_not_ready, ", ")]")
+		log_debug("Ticker: Players voted for [mode.name], but did not ready up: [jointext(voted_not_ready, ", ")]")
```

Comment:
Why debug? This should go in the normal admin or game logs.

Reply:
Admins are notified via message_admins in the ongoing round.

Its logged as debug because during analysis it provides additional context why the round did not start, and its not a relevant event that is needed for post-round administrative actions.

Reply:
It's not debugging logs though, that sounds like something that should go into the general `GAME` log. If it was just a debug log, it wouldn't list who.

Reply:
To boot, generally something shouldn't be written to both admin logs and debug logs as they both (can) show to people in-game.